xml/schema/Contribute/Contribution.xml

```diff
@@ -77,7 +77,7 @@
   </foreignKey>
   <field>
     <name>financial_type_id</name>
-    <title>Financial Type</title>
+    <title>Financial Type ID</title>
```

Comment:
Why this change?

Reply:
We have done this with a few fields - it differentiates them in search builder. I'm OK with leaving out if you want